src/world/action/walk.c

```diff
@@ -1,13 +1,125 @@
 #include "common.h"
+#include "world/actions.h"
+
+s32 D_802B6910_E23FF0[] = {0x000A0002, 0x000A002B, 0x000A002D, 0x000A002F, 0x000A0031, 0x000A0033, 0x000A0035, 0x000A0037, 0x000A0039, 0x000A003B, 0x000A003D, 0x000A003F,
```

Comment:
These are peach player animation IDs.... it would be good to add a comment to that effect so we can parameterize them later